src/services/dataFetcher.ts

```
@@ -55,11 +84,15 @@ export class DataFetcher {
           errorMessage,
         };
         storageManager.addLog({
-          level: 'error',
+          level: APILogLevel.ERROR,
           apiCall,
           reason: errorMessage,
           details: logDetails,
         });
+
+        // Update lastDataRequest even on error to prevent infinite retry loops
+        this.setTileState<T>(storageKey, null, false);
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)